scripts/world_bank/datasets/datasets.py

```diff
@@ -57,13 +77,18 @@ class Mode:
     FETCH_DATASETS = 'fetch_datasets'
     DOWNLOAD_DATASETS = 'download_datasets'
     WRITE_WB_CODES = 'write_wb_codes'
+    LOAD_STAT_VARS = 'load_stat_vars'
+    WRITE_OBSERVATIONS = 'write_observations'
 
 
 flags.DEFINE_string(
-    'mode', Mode.WRITE_WB_CODES,
-    f"Specify one of the following modes: {Mode.FETCH_DATASETS}, {Mode.DOWNLOAD_DATASETS}, {Mode.WRITE_WB_CODES}"
+    'mode', Mode.WRITE_OBSERVATIONS,
+    f"Specify one of the following modes: {Mode.FETCH_DATASETS}, {Mode.DOWNLOAD_DATASETS}, {Mode.WRITE_WB_CODES}, {Mode.LOAD_STAT_VARS}, {Mode.WRITE_OBSERVATIONS}"
 )
 
+flags.DEFINE_string('stat_vars_file', 'svs.csv',
```

Comment:
We could add the final statvars.csv used for the import into the folder.

Reply:
Yes, the final statvars is in this folder `svs.csv`.